deploy/02_deploy_reputation_contracts.js

```diff
@@ -2,10 +2,10 @@ const deployFunction = async ({ getNamedAccounts, deployments }) => {
   const { deploy } = deployments;
   const { root } = await getNamedAccounts();
 
-  await deploy("Reputation", {
+  await deploy("RatingReputation", {
     contract: "Reputation",
     from: root,
-    args: [],
+    args: ["PrimeDAO Rating Reputation", "PRR"],
```

Comment:
If we already specify the args here, for the deployment of the `Reputation` contract, did we let go of the general Prime Rating? When I read the issue it seems like we want to make the deployment of rating tokens more general. That's also what I see the changes reflect in `Reputation.sol`. I think I'm missing something but are we not sort of specifying only one way to deploy here then?

Reply:
I'm not sure, but it could be that your question on the implementation is answered by the second bullet point:

> ### What's new?
> 
>     * generalized the Reputation contract so that token name & symbol are passed as constructor args
> 
>     * adapted the existing deploy script, so that it deploys this new token. I believe since we have the previously deployed `Reputation` in the `deployments` folder, it is not super important that we preserve the original deploy script.
> 
>     * generalized the hardhat tasks in `reputationManagement.js`
> 
>     * deployed to Rinkeby (& verified): https://rinkeby.etherscan.io/address/0xa1C2eab7286dB99F481682e8165d2F1F61C4D7Ad#code
> 
> 
> closes #141

Reply:
Yes that makes it clear. Thanks

Reply:
I would have preferred to let the deploy script take the constructor params as arguments, but this was not _easily_ doable. So I could have either created an additional new script, or adapt the existing one. I opted for the latter for the reason stated above.